technical-resources/protocol-overview/smart-contract-addresses-base.md

```diff
@@ -0,0 +1,105 @@
+| WithdrawalManagerInitializer | [`0x899b7d637e35B2C57E845dB1064a4e58639D3A8D`](https://basescan.org/address/0x899b7d637e35B2C57E845dB1064a4e58639D3A8D) | [`withdrawal-manager @ v1.0.0`](https://github.com/maple-labs/withdrawal-manager/releases/tag/v1.0.0) |
+
+## Factory Contracts
+
+| Contract | Address | Commit Hash |
+| -------- | ------- | ----------- |
+| FixedTermLoanFactory | [`0x1915A8dE08A92b846dF7C845e140E4b0714820bd`](https://basescan.org/address/0x1915A8dE08A92b846dF7C845e140E4b0714820bd) | [`fixed-term-loan @ v2.0.0`](https://github.com/maple-labs/fixed-term-loan/releases/tag/v2.0.0) |
+| FixedTermLoanManagerFactory | [`0x7D75cF9Aab6cB9598bB6d9Bd81BaAA288cecA9Bf`](https://basescan.org/address/0x7D75cF9Aab6cB9598bB6d9Bd81BaAA288cecA9Bf) | [`fixed-term-loan-manager @ v1.0.0`](https://github.com/maple-labs/fixed-term-loan-manager/releases/tag/v1.0.0) |
+| LiquidatorFactory | [`0x7b0267C13B994cdb58b8ED3a65b7A09a07432A76`](https://basescan.org/address/0x7b0267C13B994cdb58b8ED3a65b7A09a07432A76) | [`liquidations @ v2.0.0`](https://github.com/maple-labs/liquidations/releases/tag/v2.0.0) |
+| OpenTermLoanFactory | [`0x8B487386E432F0f82Ca611F59dBE973761FBb1Ad`](https://basescan.org/address/0x8B487386E432F0f82Ca611F59dBE973761FBb1Ad) | [`open-term-loan @ v1.0.1`](https://github.com/maple-labs/open-term-loan/releases/tag/v1.0.1) |
+| OpenTermLoanManagerFactory | [`0xD14D0b4720bDBE03aB88A8Fd53be28c6d46426F5`](https://basescan.org/address/0xD14D0b4720bDBE03aB88A8Fd53be28c6d46426F5) | [`open-term-loan-manager @ v1.0.0`](https://github.com/maple-labs/open-term-loan-manager/releases/tag/v1.0.0) |
+| PoolManagerFactory | [`0xD4f0e224222e3F767AC389B2b1b7663990DFa6E9`](https://basescan.org/address/0xD4f0e224222e3F767AC389B2b1b7663990DFa6E9) | [`pool-v2 @ v1.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v1.0.0) |
+| WithdrawalManagerFactory | [`0x0434FC9e7D88294dAac40dDF316754B2053D613b`](https://basescan.org/address/0x0434FC9e7D88294dAac40dDF316754B2053D613b) | [`withdrawal-manager @ v1.0.0`](https://github.com/maple-labs/withdrawal-manager/releases/tag/v1.0.0) |
+
+## Active Pool Contracts
+
+### Cash Management - USDC Pool Contracts
+
+| Contract | Address | Commit Hash |
+| -------- | ------- | ----------- |
+| Pool | [`0xdd5bb9acf5e02089735a33344c6e3a8bb0d4075d`](https://basescan.org/address/0xdd5bb9acf5e02089735a33344c6e3a8bb0d4075d) | [`pool-v2 @ v1.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v1.0.0) |
+| PoolManager (Proxy) | [`0x7aa13f647691a709dc3360acad7eb758179790ee`](https://basescan.org/address/0x7aa13f647691a709dc3360acad7eb758179790ee) | [`proxy-factory @ v1.0.0`](https://github.com/maple-labs/proxy-factory/releases/tag/v1.0.0) |
+| FixedTermLoanManager (Proxy) | [`0xc3e5eec47cccdfae4bca7bf40ca2711c4cb60491`](https://basescan.org/address/0xc3e5eec47cccdfae4bca7bf40ca2711c4cb60491) | [`proxy-factory @ v1.0.0`](https://github.com/maple-labs/proxy-factory/releases/tag/v1.0.0) |
```

Comment:
As noted above, this is a v3 FTLM
https://github.com/maple-labs/fixed-term-loan-manager/releases/tag/v3.0.1